### ISSUE TITLE
No-profile mode doesn't escape commas in filenames

### ISSUE THREAD

**Issue by mistydemeo** [digital-preservation/droid#34]
DROID uses commas to separate file paths from file IDs, but doesn't perform any escaping of the filename in the case that it contains a comma. For example:

```
$ droid -Nr /tmp/id -Ns ~/.droid6/signature_files/DROID_SignatureFile_V69.xml 2>/dev/null
DROID 6.1.2 No Profile mode: Runtime Information
Resource folder: /tmp/id
Binary signature file: /Users/vlcice/.droid6/signature_files/DROID_SignatureFile_V69.xml
Container signature file: None
Extension filter: No filter set
Recurse folders: False
Open archives: False
/private/tmp/id/foo,bar,Unknown
```

This renders the output somewhat ambiguous, though it's not too severe since the separator will always be the _final_ comma.

It'd be nice if the filename's commas were escaped in this case. Since the output is vaguely CSV-like, maybe using quotes:

```
"/private/tmp/id/foo,bar",Unknown
```

**Comment by paulyoung84** [comment]
This issue has been added to the DROID backlog, it will be assigned an appropriate priority but is unlikely to be included in the next release.

**Comment by sparkhi** [comment]
This is fixed in 6.6.0
